### PR TITLE
Add helm charts

### PR DESCRIPTION
**Pull request by yaocw2020**
Add helm charts:
- harvester-load-balancer:0.1.1
- harvester-cloud-provider:0.1.1
- harvester-csi-driver:0.1.0

issue: https://github.com/harvester/harvester/issues/1019

### CODE REVIEW

**Comment by yaocw2020** [comment]
dependency:
- harvester-cloud-provider v0.1.1 release
- harvester-load-balancer v0.1.1 release
- harvester-csi-driver v0.1.0 release

File: charts/harvester-load-balancer/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.1
```
Comment (gitlawr) [inline charts/harvester-load-balancer/Chart.yaml:18]: Have we released 0.1.0?

Comment (yaocw2020) [reply charts/harvester-load-balancer/Chart.yaml:18]: Yes. I have cut 0.1.0 version before.

**Comment by gitlawr** [comment]
Can you add the systemDefaultRegistry options like https://github.com/rancher/charts/blob/dev-v2.6/charts/longhorn/longhorn/1.1.100/values.yaml#L4-L6

It is a rancher global setting that will be passed to the chart release.

**Pull request closed without merging**